### ISSUE TITLE
Password Protected Sites

### ISSUE THREAD

**Issue by joshpatton** [wummel/linkchecker#673]
Hello, I'm a bit of a noob but I can't seem to get it to scan my site that is password protected. I entered the values in the config file and removed the comment tag and yet it's still not working. Is there a guide someone can point me to for doing this? Maybe I'm doing something incorrectly. Thanks!

**Comment by kaalen** [comment]
Yeah, I'm having the same problem. Looks like the config file settings either don't work or I'm not setting it up right. Tips would be appreciated.

**Comment by cmeihsl** [comment]
Same for me, tried many different settings in the linkcheckerrc file, on several sites, could not get it to log in.

**Comment by peknis01** [comment]
Is there a way to check password protected sites using the GUI? Works fine in the command line with -u <user> -p , then enter the password.

**Comment by dpalic** [comment]
Thank you for the issue report. Sadly this project is dead, and a new team is around with https://github.com/linkcheck/linkchecker 
for more details please see: #708
**Also please close this issue** and report it freshly on the new repo https://github.com/linkcheck/linkchecker/issues
